server/ui.go

```diff
@@ -50,19 +48,37 @@ func newUIOption(frontendAddr string, uiIP string, uiPort int, configDir string)
 	return WithUI(uiserver.NewServer(uiserveroptions.WithConfigProvider(cfg))), nil
 }
 
-func NewUIConfig(frontendAddr string, uiIP string, uiPort int, configDir string) (*uiconfig.Config, error) {
+func NewUIConfig(c *uiConfig, configDir string) (*uiconfig.Config, error) {
 	cfg := &uiconfig.Config{
-		Host: uiIP,
-		Port: uiPort,
+		Host:                c.Host,
+		Port:                c.Port,
+		TemporalGRPCAddress: c.TemporalGRPCAddress,
+		EnableUI:            c.EnableUI,
+		Codec: uiconfig.Codec{
+			Endpoint: c.CodecEndpoint,
```

Comment:
Is it okay if this is empty?

Reply:
yea the default value is empty. This means it's disabled